.gitignore

```diff
@@ -5,3 +5,5 @@ cmake-build-release/
 build/
 venv/
 **/.pytest_cache/
+**/__pycache__/
+*.db3-*
```

Comment:
Not quite. There's a `-` at the end of the `.db3-` (line 9). This is to exclude the `.db3-shm` and `.db3-wal` files that get generated during the testing process.

If you prefer, though, I could change line 9 into two lines:
`.db3-shm`
`.db3-wal`
I just did it this way since it was less typing.

Reply:
Looking at the diff - you have checked in at least one each of `.db3-shm` and `.db3-wal` files

Reply:
Good catch! Hopefully they're removed now...